devops/repos/update_repo.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+source /etc/profile
+set -e 
+
+DEVOPS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+SCRIPT_DIR=${SCRIPT_DIR:-../../install/scripts/}
+SCRIPT_DIR_ABS=$DEVOPS_DIR/$SCRIPT_DIR
+REPO_BASE_DIR=${REPO_BASE_DIR:-/data/repos/}
+
+BUILD_HIPSYCL=${BUILD_HIPSYCL:-ON}
+BUILD_CUDA=${BUILD_CUDA:-OFF}
+BUILD_ROCM=${BUILD_ROCM:-OFF}
+BUILD_BASE=${BUILD_BASE:-OFF}
+NO_BUILD=${NO_BUILD:-OFF}
+
+HIPSYCL_REPO_USER=${HIPSYCL_REPO_USER:-illuhad}
+HIPSYCL_REPO_BRANCH=${HIPSYCL_REPO_BRANCH:-master}
+
+# Default is LLVM 10
+LLVM_9_BUILD=${LLVM_9_BUILD:-ON}
+CONTAINER_DIR_SUFFIX=${CONTAINER_DIR_SUFFIX:-containers}
+CONTAINER_DIR=$SCRIPT_DIR_ABS/$CONTAINER_DIR_SUFFIX
+[ "$LLVM_9_BUILD" = "ON" ] && CONTAINER_DIR=$SCRIPT_DIR_ABS/clang9-$CONTAINER_DIR_SUFFIX
+
+export CONTAINER_DIR
+export BUILD_HIPSYCL
+export BUILD_CUDA
+export BUILD_ROCM
+export BUILD_BASE
+export NO_BUILD
+export LLVM_9_BUILD
```

Comment:
Rather than querying for LLVM 9 explicitly, wouldn't it be more beneficial to have the llvm branch as an environment variable? Otherwise, we may have to change the code here once LLVM 9 is obsolete and the discussion is whether to use LLVM 10 or 11.

Reply:
Thank you, This is a good point. I will change it to be so.